src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -11469,6 +11694,12 @@ public sealed partial class NeutralResourcesLanguageAttribute : System.Attribute
         public string CultureName { get { return default(string); } }
         public System.Resources.UltimateResourceFallbackLocation Location { get { return default(System.Resources.UltimateResourceFallbackLocation); } }
     }
+    internal partial class PRIExceptionInfo
```

Comment:
All the internal types were added to support different scenarios, mostly the WinRT scenarios where System.Runtime.WindowsRuntime depended on internal types in mscorlib so to keep those functioning we needed these so the type-forwards existed in mscorlib.